journal/modernized/08-chapter-8.adoc

```diff
@@ -286,15 +288,13 @@ yet not I,
 but Christ liveth in me,`" and the life I now live is not according to the flesh,
 but in my measure according to the blessed will of the dear Son of God,
 and to Him is the praise and thanksgiving, now and forever.
-Amen.
-
-Written by myself who am almost blind, J. C.
+Amen. Written by myself who am almost blind, John Conran.
```

Comment:
I think we should switch the Amen back to being joined to the paragraph above, so this new line should start with `Written by`...